src/style.css

```diff
@@ -19,17 +23,18 @@ h2 {
 }
 
 .title {
-  font-size: 3em;
+  font-size: 5em;
   padding-top: 2em;
-  padding-left: 6em;
+  padding-left: 3em;
+  color: gold;
 }
 
 .container {
   display: flex;
   align-items: center;
   justify-content: center;
   padding: 5em;
-  height: 80vh;
+  height: 100%;
```

Comment:
- [ ] Kindly adjust the padding between the title and the other sections to match the wireframe.
- Your design
![leaderboard](https://user-images.githubusercontent.com/46851023/152636995-1cdb360a-8fd8-4a2f-8f99-e30987ab229e.png)
- Wireframe
![wireframe](https://user-images.githubusercontent.com/46851023/152637003-a5189d39-8fd2-482c-9367-2da4d29db28a.png)